CMakeLists.txt

```diff
@@ -27,17 +27,19 @@ list( APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake )
 project( nest CXX C )
 set( NEST_USER_EMAIL "users@nest-simulator.org" )
 
+include( ColorMessages )
+
 # check if the given CMAKE_INSTALL_PREFIX is not empty
 if("${CMAKE_INSTALL_PREFIX}" STREQUAL "")
-  message(FATAL_ERROR "CMAKE_INSTALL_PREFIX cannot be an empty string")
+  printERROR("CMAKE_INSTALL_PREFIX cannot be an empty string")
```

Comment:
```suggestion
  printError("CMAKE_INSTALL_PREFIX cannot be an empty string")
```

Reply:
According to [the CMake documentation](https://cmake.org/cmake/help/latest/command/message.html), using `FATAL_ERROR` does not just print the message, but also stops all processing and generation.

As much as I always disliked that side effect, I would not want the behavior changed by this PR. Could you please add a call like `message(FATAL_ERROR "")` to the definition of `printError`?

Or even better (and more explicit) wrap that call into a function `exit_cmake()` and calls to that everywhere right after where we use `printError`?

Reply:
Actually, the `printError` will eventually call the function `message` with `FATAL_ERROR`, or am I misunderstanding your comment?